packages/cozy-clisk/src/contentscript/ContentScript.js

```diff
@@ -425,10 +425,25 @@ export default class ContentScript {
    * Send log message to the launcher
    *
    * @param {string} message : the log message
```

Comment:
* @param {"debug"|"info"|"warn"|"error"} level: the log level
🙏

Reply:
Fixed